server/base/src/main/java/org/apache/accumulo/server/util/ListVolumesUsed.java

```diff
@@ -111,42 +112,28 @@ private static void listTable(String name, ServerContext context) throws Excepti
 
     volumes.clear();
 
-    scanner.clearColumns();
-    scanner.setRange(MetadataSchema.DeletesSection.getRange());
-
-    for (Entry<Key,Value> entry : scanner) {
-      String delPath = entry.getKey().getRow().toString()
-          .substring(MetadataSchema.DeletesSection.getRowPrefix().length());
-      volumes.add(getTableURI(delPath));
-    }
-
     System.out.println("Listing volumes referenced in " + name
         + " deletes section (volume replacement occurrs at deletion time)");
 
-    for (String volume : volumes) {
-      System.out.println("\tVolume : " + volume);
+    Iterator<String> delPaths = context.getAmple().getGcCandidates(name, "");
+    while (delPaths.hasNext()) {
+      System.out.println("\tVolume : " + getTableURI(delPaths.next()));
     }
 
-    volumes.clear();
+    System.out.println("Listing volumes referenced in " + name + " current logs");
 
     WalStateManager wals = new WalStateManager(context);
     for (Path path : wals.getAllState().keySet()) {
-      volumes.add(getLogURI(path.toString()));
-    }
-
-    System.out.println("Listing volumes referenced in " + name + " current logs");
-
-    for (String volume : volumes) {
-      System.out.println("\tVolume : " + volume);
+      System.out.println("\tVolume : " + getLogURI(path.toString()));
     }
   }
 
   public static void listVolumes(ServerContext context) throws Exception {
     listZookeeper(context);
     System.out.println();
-    listTable(RootTable.NAME, context);
+    listTable(Ample.DataLevel.METADATA, context);
```

Comment:
Its unrelated, but this code should also do `listTable(Ample.DataLevel.ROOT, context);` now.  Also the specialized zookeeper code could go away.   I can open a follow on issue.

Reply:
Opened #1350